src/legacy/ui/public/management/components/sidebar_nav.test.ts

```diff
@@ -77,6 +77,7 @@ const managementSections = [
 
 describe('Management', () => {
   it('filters and filters and maps section objects into SidebarNav items', () => {
+    // @ts-ignore
```

Comment:
This was giving me some trouble so I decided to ignore it. I'm happy to revisit with some help. Should be addressed in my next PR since there will be a layer of abstraction to merge two different section formats (new and old platforms)